pkg/store/kotsstore/supportbundle_store.go

```diff
@@ -212,53 +212,6 @@ func (s *KOTSStore) ListSupportBundles(appID string) ([]*types.SupportBundle, er
 	return supportBundles, nil
 }
 
-func (s *KOTSStore) DeletePendingSupportBundle(id string) error {
-	clientset, err := k8sutil.GetClientset()
-	if err != nil {
-		return errors.Wrap(err, "failed to get clientset")
-	}
-
-	if err := clientset.CoreV1().Secrets(os.Getenv("POD_NAMESPACE")).Delete(context.TODO(), fmt.Sprintf("pendingsupportbundle-%s", id), metav1.DeleteOptions{}); err != nil {
-		if kuberneteserrors.IsNotFound(err) {
-			return nil
-		}
-		return errors.Wrap(err, "failed to delete")
-	}
-
-	return nil
-}
```

Comment:
No more "PendingSupportBundle" secrets. Just support bundles that are "inProgress"